### PR TITLE
ED-1756 create standalone SSO/great.gov.uk account

### PR DESCRIPTION
**Pull request by kowalcj0**
This implements 2 scenarios:

```gherkin

    @ED-1756
    @sso
    @account
    Scenario: Suppliers should receive an email verification msg after creating a standalone SSO/great.gov.uk account
      Given "Peter Alder" is an unauthenticated supplier

      When "Peter Alder" creates a SSO/great.gov.uk account

      Then "Peter Alder" should be told about the verification email
      And "Peter Alder" should receive an email verification msg entitled "Your great.gov.uk account: Please Confirm Your E-mail Address"


    @ED-1756
    @sso
    @account
    Scenario: Suppliers should be able to confirm email address for a standalone SSO/great.gov.uk account
      Given "Peter Alder" created a standalone SSO/great.gov.uk account with unverified email address

      When "Peter Alder" decides to confirm her email address by using the email confirmation link
      And "Peter Alder" confirms the email address for SSO/great.gov.uk account

      Then "Peter Alder" should be on Welcome to your great.gov.uk profile page
      And "Peter Alder" should be signed in to SSO/great.gov.uk account
```

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -71,12 +71,12 @@ PYTEST_ARGS :=
...
 SET_PYTEST_ENV_VARS := \
 	export API_CLIENT_KEY=debug; \
-	export DIRECTORY_API_URL=http://directory-api-dev.herokuapp.com/; \
-	export DIRECTORY_BUYER_API_URL=http://dev.buyer.directory.uktrade.io/; \
-	export DIRECTORY_SSO_URL=http://www.dev.sso.uktrade.io/; \
-	export DIRECTORY_PROFILE_URL=http://www.dev.profile.uktrade.io; \
-	export DIRECTORY_UI_BUYER_URL=http://www.dev.buyer.directory.uktrade.io/; \
-	export DIRECTORY_UI_SUPPLIER_URL=http://www.dev.supplier.directory.uktrade.io/; \
+	export DIRECTORY_API_URL=http://directory-api-dev.herokuapp.com; \
+	export DIRECTORY_BUYER_API_URL=http://dev.buyer.directory.uktrade.io; \
```
Comment (konradko) [inline makefile]: you can use https

Comment (kowalcj0) [inline makefile]: I could, but sometimes we're getting `http` in the 302 redirect responses.

Comment (kowalcj0) [inline makefile]: maybe it'd be worth investigating, whether FAB, FAS, SSO & Profile always return the correct protocol.

Comment (konradko) [inline makefile]: sounds like it, please add a task

**Pull request merged**